.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -31,6 +31,7 @@ locals {
   frontend_image_repo          = local.secret["ecrs"]["frontend"]["url"]
   backend_image_repo           = local.secret["ecrs"]["backend"]["url"]
   upload_image_repo            = local.secret["ecrs"]["processing"]["url"]
+  lambda_upload_success_repo   = local.secret["ecrs"]["upload_success"]["url"]
```

Comment:
ECR repo for the new lambda's Docker image (added in https://github.com/chanzuckerberg/single-cell-infra/pull/474/files)